ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -259,7 +263,7 @@
      (doall
       (map-indexed
        (fn [i option]
-         (let [checked? (selected option)]
+         (let [checked? (if (nil? (selected option)) false true)]
```

Comment:
you can just coerce to boolean: `(boolean (selected option))`

Reply:
changed